### PR TITLE
2dc: add config description

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add description of how to configure placement rules

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by disksing** [comment]
@nolouch Please take a look :)

File: two-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -156,6 +156,12 @@ cat rule.json
 ]
 ```
 
+你可以使用如下命令使用 rule.json 中的配置来覆盖原有的设置：
```
Comment (en-jin19) [inline two-data-centers-in-one-city-deployment.md:159]:
```suggestion
你可以使用如下命令使用 `rule.json` 中的配置来覆盖原有的设置：
```

File: two-data-centers-in-one-city-deployment.md [hunk 2]
```diff
@@ -156,6 +156,12 @@ cat rule.json
...
+
+```bash
+pd-ctl config placement-rules rule-bundle save --in="rule.json"
+```
```
Comment (en-jin19) [inline two-data-centers-in-one-city-deployment.md:163]:
```suggestion
你可以使用如下命令使用 rule.json 中的配置来覆盖原有的设置：

{{< copyable "shell-regular" >}}

```bash
pd-ctl config placement-rules rule-bundle save --in="rule.json"
```
```

File: two-data-centers-in-one-city-deployment.md [hunk 3]
```diff
@@ -156,6 +156,12 @@ cat rule.json
 ]
 ```
 
+你可以使用如下命令使用 rule.json 中的配置来覆盖原有的设置：
+
```
Comment (en-jin19) [inline two-data-centers-in-one-city-deployment.md:160]:
```suggestion
如果需要使用 `rule.json` 中的配置，你可以通过如下命令覆盖原有的配置：

{{< copyable "shell-regular" >}}

```

File: two-data-centers-in-one-city-deployment.md [hunk 4]
```diff
@@ -156,6 +156,13 @@ cat rule.json
 ]
 ```
 
+如果需要使用 `rule.json` 中的配置，你可以通过如下命令覆盖原有的配置：
+
+{{< copyable "shell-regular" >}}
```
Comment (en-jin19) [inline two-data-centers-in-one-city-deployment.md:161]:
```suggestion
{{< copyable "shell-regular" >}}

```

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by JmPotato — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**